# General parameters
package_name_param: 1

# Dynamic reconfigure parameters
int_param: 50
double_param: 0.5
str_param: "Hello World"
bool_param: true
